MaterialDesignThemes.Wpf/CustomBaseColorTheme.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Windows;
+using System.Windows.Media;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public class CustomBaseColorTheme : ResourceDictionary, IBaseTheme
```

Comment:
Should this class be deriving from the existing `CustomColorTheme` class?

Reply:
Well, I'm affraid on it, because `CustomColorTheme` (that's unnecessary from final ResourceDictionary) has an BaseTheme property and It could maybe generates a wrong XAML ResourceDictionary file:

![image](https://user-images.githubusercontent.com/15158923/92304108-89b5bd80-ef51-11ea-979b-14b9e4e55265.png)

However, I am completely open to evaluate this subject.